dvc/config_schema.py

```diff
@@ -329,4 +326,8 @@ def __call__(self, data):
         "config_dir": str,
         "config_name": str,
     },
+    "studio": {
+        "token": str,
+        "url": str,
```

Comment:
Maybe this should be studio.base_url? Any opinions @dberenbaum?

Reply:
At the moment, it’ll be something like this:
```console
dvc config studio.url https://dev.iterative.ai
```

I find url simpler, but I worry if we should reserve url for other purposes (eg: project url if needed in the future).

Reply:
Thanks for the explanation. I'd vote to keep it URL then, since projects or other URL parts could each be their own options without the full URL (like `dvc config studio.project example-get-started-experiments-hktakvv83z`), but it's not a strong opinion or blocker.